### PR TITLE
Add json model

### PR DESCRIPTION
**Pull request by jinSasaki**
I added Bucketeer2 target to the existing Xcode project. I plan to move to or replace Bucketeer when merging to master.

Models are referenced by Android JSON models. https://github.com/ca-dp/bucketeer-android-sdk/tree/v2/bucketeer2/src/main/kotlin/io/bucketeer/sdk/android/internal/model

### CODE REVIEW

**Comment by cre8ivejp** [comment]
Could you please remove Xcode's generated comments on the top of each new file?

E.g.

```swift 
//
//  Bucketeer2.h
//  Bucketeer2
//
//  Created by Jin Sasaki on 2022/09/07.
//  Copyright © 2022 Bucketeer. All rights reserved.
//
```

**Comment by jinSasaki** [comment]
OK. I'll remove the comments.

**Comment by jinSasaki** [comment]
Removed https://github.com/ca-dp/bucketeer-ios-sdk/pull/82/commits/60625f7a7753f3dbef014fa104425782b831003e

**Review by cre8ivejp — APPROVED** [review]
Thank you!

File: Bucketeer2/Sources/Internal/JSON/JSON.UserEvaluationsState.swift [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import Foundation
+
+extension JSON {
+    enum UserEvaluationsState: Int {
```
Comment (cre8ivejp) [inline Bucketeer2/Sources/Internal/JSON/JSON.UserEvaluationsState.swift:4]: Sorry for mentioning this only now, but the `UserEvaluationsState` became deprecated. The server is always returning FULL, so you can remove this.
I will remove this value from the API response next week. I have also asked to remove it in the android SDK.

Comment (jinSasaki) [reply Bucketeer2/Sources/Internal/JSON/JSON.UserEvaluationsState.swift]: OK! I removed `UserEvaluationState`. 🔥  https://github.com/ca-dp/bucketeer-ios-sdk/pull/82/commits/471fd704f23ca220e1f09e9726eb79dc69c82fb0

Comment (cre8ivejp) [reply Bucketeer2/Sources/Internal/JSON/JSON.UserEvaluationsState.swift]: Thank you!

**Pull request merged**